etc/chat-core.api.md

```diff
@@ -50,6 +48,14 @@ export enum Environment {
     SANDBOX = "SANDBOX"
 }
 
+// Warning: (ae-internal-missing-underscore) The name "InternalConfig" should be prefixed with an underscore because the declaration is marked as @internal

```

Comment:
not sure if we want to follow this rule -- If not, lets update api extractor to ignore this rule

Reply:
Yeah I'm not a fan of it for interfaces, that seems unnecessary. Updating.